### ISSUE TITLE
Misleading label for query duration: "duration_μs"

### ISSUE THREAD

**Issue by holtkamp** [abacaphiliac/doctrine-psr-sql-logger#5]
Currently the label used for query duration is `duration_μs`: 

https://github.com/abacaphiliac/doctrine-psr-sql-logger/blob/539855787a8d679e561d9b5e3c3a1ab0eac5e20e/src/PsrSqlLogger.php#L70

For me `μs` suggests that the value is indicated in microseconds, also see https://en.wikipedia.org/wiki/Microsecond

However, the value itself is indicated in seconds, only the *level of detail* is in microseconds. 

So I would expect the label to be: `duration_s`, or simply `duration`...

Maybe a bit nitpicking 😉 , but it might be something to think about.

**Comment by abacaphiliac** [comment]
good catch. i must have misunderstood the `microtime` docs. i'm definitely interested in getting a quick fix in. is that something you'd be interested in contributing as well? i'm also interested in getting a good test around the duration units to not only confirm the desired behavior now but also to prevent any mistakes in the future, haha. any ideas on that front?

**Comment by holtkamp** [comment]
> i'm definitely interested in getting a quick fix in. is that something you'd be interested in contributing as well

Well, I can open a PR to replace `duration_μs` with `duration`, or `durationInSeconds`. We should probably not over-engineer this 😄  

> i'm also interested in getting a good test around the duration units to not only confirm the desired behavior now but also to prevent any mistakes in the future, haha. any ideas on that front?

mmm, interesting, never thought about it like that, but we "could" consider using a [`DateInterval`](https://php.net/manual/en/class.dateinterval.php) representation, as of PHP 7.1 this supports the fractional part.

```php
$d1=new DateTime("2012-07-08 11:14:15.638276"); 
$d2=new DateTime("2012-07-08 11:14:15.889342"); 
$diff=$d2->diff($d1); 
print_r( $diff ) ;
```

[results in](https://3v4l.org/LAIP6)

```PHP
DateInterval Object
(
    [y] => 0
    [m] => 0
    [d] => 0
    [h] => 0
    [i] => 0
    [s] => 0
    [f] => 0.251066
    [weekday] => 0
    [weekday_behavior] => 0
    [first_last_day_of] => 0
    [invert] => 1
    [days] => 0
    [special_type] => 0
    [special_amount] => 0
    [have_weekday_relative] => 0
    [have_special_relative] => 0
)
```

Only the question then still stands: how to "represent" this, `PT0.251066F` seems not a valid [interval specification](https://www.php.net/manual/en/dateinterval.construct.php). I think it would be best to simply put it in number of seconds, also facilitating analysis tools to analyse log files and extract query times.

**Comment by abacaphiliac** [comment]
> We should probably not over-engineer this 😄

i agree.

regarding the new label, i'd prefer some indication of units instead of simply `duration`, and the rest of the logger properties use snake-case so i'd prefer to stick with that convention. `duration_s` is acceptable but maybe we should compromise with `duration_in_seconds`, since we all know abbreviations are horrid : P

your DateInterval idea is very interesting to me. i would prefer to move to objects for managing date math and formatting anyway and we can certainly do that later. currently, this lib can be installed on php 7.0 so i'll schedule an overdue routine version bump and look into it at that time.

regarding interval formatting, there is a format function which i think we can use to reproduce the `microtime` format.

e.g.

```
php > $start = new DateTime();
php > // wait some time...
php > $start = new DateTime();
php > echo $end->diff($start)->format('%s.%f');
9.50798
```

**Comment by abacaphiliac** [comment]
resolved by #11
